lib/membrane/aac/parser.ex

```diff
@@ -43,28 +62,27 @@ defmodule Membrane.AAC.Parser do
   end
 
   @impl true
-  def handle_stream_format(
-        :input,
-        %AAC{encapsulation: encapsulation} = stream_format,
-        _ctx,
-        %{in_encapsulation: encapsulation} = state
-      ) do
-    {[stream_format: {:output, %{stream_format | encapsulation: state.out_encapsulation}}], state}
-  end
-
-  @impl true
-  def handle_stream_format(:input, %AAC{} = stream_format, _ctx, state),
-    do:
+  def handle_stream_format(:input, %AAC{} = stream_format, _ctx, state) do
```

Comment:
I would consider removing `in_encapsulation` option and inferring it from the stream format (and raising in case it changes throughout the stream), just as we do in case of H264 parser, WDYT?

Reply:
We could also rewrite `in_encapsulation` as the `out_encapsulation` in case it's not explicitly provided via the options (another thing we do in case of H264 parser)

Reply:
I was thinking about it, but decided to hold on and focus on the config feature

Reply:
maybe it would be more suitable for another PR?

Reply:
I'm wondering whether there is any use case for parser with incoming `Membrane.RemoteStream`, `in_encapsulation: :none` and `out_encapsulation: :none`, because this is the only case where assuming that when incoming stream format is `Membrane.RemoteStream` the encapsulation is `:ADTS` would be wrong, but i think there is no such use case